solidus_globalize.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'deface', '~> 1.5'
   s.add_dependency 'friendly_id-globalize'
   s.add_dependency 'globalize', '~> 6.0'
-  s.add_dependency 'solidus_core', ['>= 2.0.0', '< 4']
+  s.add_dependency 'solidus_core', ['>= 2.0.0', '< 5']
```

Comment:
should we bump the minimum version to the oldest one currently supported?

Reply:
It's not a real issue, it's more that I know we won't have the resources to be consistent with this approach at the moment.